exo-go/src/application/runner.go

```diff
@@ -77,18 +77,22 @@ func (r *Runner) getEnv() []string {
 	return formattedEnvVars
 }
 
-func (r *Runner) runImages(imageNames []string, imageOnlineTexts map[string]string, identifier string) error {
+func (r *Runner) runImages(imageNames []string, imageOnlineTexts map[string]string, identifier string) (string, error) {
+	if len(imageNames) == 0 {
+		r.logChannel <- fmt.Sprintf("all %s online", identifier)
+		return "", nil
+	}
```

Comment:
I'd prefer we avoid calling runImages with no imageNames instead of having the logic in this function. Nothing came online, so seems wrong to print out "all ... online"